docs/articles/samples/IntroPowerPlan.md

```diff
@@ -4,7 +4,12 @@ uid: BenchmarkDotNet.Samples.IntroPowerPlan
 
 ## Sample: IntroPowerPlan
 
-This sample shows how we can turn off changing power plans.
+This sample shows how we can manipulate with power plans. In BenchmarkDotNet we could change power plan in two ways. The first one is to set one from the list:
+* PowerSaver, guid: a1841308-3541-4fab-bc81-f71556f20b4a
+* Balanced, guid: 381b4222-f694-41f0-9685-ff5bb260df2e
+* High-Performance, guid: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c (the default one)
+* UserPowerPlan (a current power plan set in computer)
+The second one rely on guid string. We could easily found currently set guids with cmd command *powercfg /list*. If we set power plans in two ways at the same time, the second one will be used.
```

Comment:
nit: you can put the console command in a separate block:

````
```cmd
powercfg /list
```
````

Then it's going to be displayed as:

```cmd
powercfg /list
```

Reply:
Funny, on my PC (`Windows 10.0.17763.437 (1809/October2018Update/Redstone5)`)

I have an additional Power Plan called Ultimate Performance. I wonder what it is any why you don't have it.

```log
Power Scheme GUID: 381b4222-f694-41f0-9685-ff5bb260df2e  (Balanced)
Power Scheme GUID: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c  (High performance) *
Power Scheme GUID: a1841308-3541-4fab-bc81-f71556f20b4a  (Power saver)
Power Scheme GUID: e9a42b02-d5df-448d-aa00-03f14749eb61  (Ultimate Performance)
```

Reply:
> 
> 
> Why are you using `string` instead of `Guid` here? Does `Characteristic` does not support it?

Unfortunately, if we used Guid here, we would receive compile error, so I decided to leave string here. In .notcs file it looks like:
As a GUID: BenchmarkDotNet.Jobs.PowerPlanMode.PowerPlanGuidCharacteristic[job] = e9a42b02-d5df-448d-aa00-03f14749eb61;
As a string: BenchmarkDotNet.Jobs.PowerPlanMode.PowerPlanGuidCharacteristic[job] = $@"e9a42b02-d5df-448d-aa00-03f14749eb61";

> 
> 
> Funny, on my PC (`Windows 10.0.17763.437 (1809/October2018Update/Redstone5)`)
> 
> I have an additional Power Plan called Ultimate Performance. I wonder what it is any why you don't have it.
> 
> ```
> Power Scheme GUID: 381b4222-f694-41f0-9685-ff5bb260df2e  (Balanced)
> Power Scheme GUID: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c  (High performance) *
> Power Scheme GUID: a1841308-3541-4fab-bc81-f71556f20b4a  (Power saver)
> Power Scheme GUID: e9a42b02-d5df-448d-aa00-03f14749eb61  (Ultimate Performance)
> ```
It seems that Ultimate Performance was added in some Windows 10 update. The rest of the power plans have existed since the Windows Vista. It is easy to add it to BenchmarkDotNet. However, for systems which do not have that plan, we receive an error message  "Cannot set power plan..." and stay with the user power plan. What is your opinion? Should we add Ultimate Performance?

There is another thing that wonders me. Although I don't have that power plan, I see the same version of OS `Windows 10.0.17763.437 (1809/October2018Update/Redstone5). It is quite strange that we have the same OS version and we vary in power plans. Perhaps, it is okay. What are your thoughts about it?

Reply:
Could you please add it to the list of supported power plans? But keep `High performance` the default one?